elements/designer-mini-nav/designer-mini-nav.html

```diff
@@ -47,6 +47,46 @@
 
   Polymer({
     is: 'designer-mini-nav',
+
+    properties: {
+
+      expanded: {
+        type: Boolean,
+        value: false,
+        reflectToAttribute: true,
+      },
+
+      _collapsing: {
+        type: Boolean,
+        value: function() { return false; },
+      },
+
+    },
+
+    listeners: {
+      'designer-mini-nav-collapse': '_onCollapse',
```

Comment:
Sometimes elements cooperate. I could try to make a more abstract event or marker for the element so that the mini-nav can listen for something generic, but I think that would be generality for generality sake right now. This just works.